selfdrive/test/process_replay/migration.py

```diff
@@ -2,16 +2,58 @@
 
 from cereal import messaging
 from openpilot.selfdrive.test.process_replay.vision_meta import meta_from_encode_index
+from openpilot.selfdrive.car.toyota.values import EPS_SCALE
+from panda import Panda
 
 
-def migrate_all(lr, old_logtime=False, camera_states=False):
+def migrate_all(lr, old_logtime=False, panda_states=False, camera_states=False):
   msgs = migrate_sensorEvents(lr, old_logtime)
   msgs = migrate_carParams(msgs, old_logtime)
+  if panda_states:
+    msgs = migrate_pandaStates(msgs, old_logtime)
   if camera_states:
     msgs = migrate_cameraStates(msgs)
 
   return msgs
 
+def migrate_pandaStates(lr, old_logtime=False):
+  all_msgs = []
+  # TODO: safety param migration should be handled automatically
+  safety_param_migration = {
```

Comment:
@sshane is this still needed?

Reply:
Can't regen the routes here to test (missing `wideCameraState`), but I'm pretty sure. controlsd checks that the safety params match, which will be different with our newer flags or EPS scaling factors.

Reply:
FYI https://github.com/commaai/openpilot/pull/28829 removed the camera replay function that allowed old segments to work

Reply:
For the wide cam: https://github.com/commaai/openpilot/pull/30143